client/my-sites/stats/stats-purchase/styles.scss

```diff
@@ -134,6 +134,41 @@ $button-padding: 8px 32px;
 	}
 }
 
+.stats-tier-upgrade-slider {
+	margin: 2px;
+	padding: 10px;
+	background: pink;
+
+	.jp-components-pricing-slider__thumb {
+		border-radius: 24px;
+		border-color: #008710;
+		background-color: #008710;
+		color: white;
+	}
+}
+.stats-tier-upgrade-slider__info-message {
+	color: green;
```

Comment:
Let's pull the colours from here and the hex values above to variables. It will be easier to update.